Streetcode/Streetcode.BLL/Mapping/Partners/PartnerProfile.cs

```diff
@@ -13,5 +14,9 @@ public PartnerProfile()
             .ForPath(dto => dto.TargetUrl.Href, conf => conf.MapFrom(ol => ol.TargetUrl));
         CreateMap<Partner, CreatePartnerDTO>().ReverseMap();
         CreateMap<Partner, PartnerShortDTO>().ReverseMap();
+        CreateMap<StreetcodePartner, PartnersUpdateDTO>()
+            .ForPath(dto => dto.PartnerId, conf => conf.MapFrom(ol => ol.PartnerId))
+            .ForPath(dto => dto.StreetcodeId, conf => conf.MapFrom(ol => ol.StreetcodeId));
+        CreateMap<PartnersUpdateDTO, StreetcodePartner>().ReverseMap();
```

Comment:
you don`t need to write the path explicitly if the name of the property is the same, in that way, you can minimize time for writing code.